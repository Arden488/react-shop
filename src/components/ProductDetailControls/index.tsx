import * as React from 'react';

import './styles.css';

const ProductDetailControls = () => (
  <div className="product-controls">
    <button className="b-btn">Add to cart</button>
  </div>
);

export default ProductDetailControls;
