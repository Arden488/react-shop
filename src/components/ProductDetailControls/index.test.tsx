import { shallow } from 'enzyme';
import * as React from 'react';

import ProductDetailControls from './index';

let component: React.ReactNode = null;

beforeEach(() => {
  component = shallow(<ProductDetailControls />);
});

it('renders correctly', () => {
  expect(component).toMatchSnapshot();
});
