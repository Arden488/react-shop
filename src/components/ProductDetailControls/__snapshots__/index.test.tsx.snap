// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="product-controls"
>
  <button
    className="b-btn"
  >
    Add to cart
  </button>
</div>
`;
